packages/babel-plugin-transform-classes/src/transformClass.ts

```diff
@@ -399,22 +399,22 @@ export default function transformClass(
       );
     }
 
-    const bareSupers = new Set<NodePath<t.CallExpression>>();
+    const bareSupers = new Array<NodePath<t.CallExpression>>();
```

Comment:
Nit:
```suggestion
    const bareSupers: NodePath<t.CallExpression>[] = [];
```
will be shorter.

Reply:
👌 done